lib/dom/layer.js

```diff
@@ -795,6 +823,16 @@
             return changes;
         }
     };
+
+    ShapeLayer.prototype.toRaw = function () {
+        var baseLayer = BaseLayer.prototype.toRaw.call(this);
+        var layer = Raw.toRaw(this, [
```

Comment:
i am wondering if this is worth exposing this as a function on the layer/shapelayer etc. that simply returns ["",""] this is a good first step to generalising these functions.